src/Components/App/App.tsx

```diff
@@ -24,14 +26,24 @@ const App: React.FC = () => {
   }
 
   return (
-    <div className="backGround">
-      <NavBar retrieveBooks={retrieveBooks} />
-      <main>
-        <h1 className="project-title">A Novel Idea</h1>
-        <BookCardContainer allBooks={books} oneBook={retrieveSingleBook}/>
-          <BookDetails />
-      </main>
-    </div>
+        <div className="backGround">
+          <main>
+            <h1 className="project-title">A Novel Idea</h1>
+            <Route exact path='/'>
+              <Home />
+            </Route>
+            <Route exact path='/books'>
+              <BookCardContainer allBooks={books} oneBook={retrieveSingleBook}/>
+              <NavBar retrieveBooks={retrieveBooks} />
+            </Route>
+            <Route exact path='/bookDetails'>
+              <BookDetails />
+            </Route>
+            <Route exact path='/error'>
+              <Error />
+            </Route>
```

Comment:
Consider adding a `Switch` to App so that you can handle any bad path that a user types in. i.e. if they try to go to `localhost:3000/potatoes`, they would get an error screen that says something like "That page does not exist, try going back home" with a button to go to the homepage.